articles/changes.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: default
+title: Changes between ROS 1 and ROS 2
+permalink: articles/changes.html
+abstract:
+  This article provides an overview about the changes being made in ROS 2 compared to ROS 1.
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## Preface
+
+Each change is described as briefly as possible but giving enough context and rational for a reader familiar with ROS 1.
+
+
+## Platforms and dependencies
+
+### Platforms
+
+ROS 1 is only being CI tested on Ubuntu.
+It is actively supported by the community on other Linux flavors as well as OS X.
+
+ROS 2 will be CI tested and supported on Ubuntu, OS X as well as Windows.
+
+
+### Languages
+
+#### C++ standard
+
+ROS 1 is targeting C++03.
+ROS 2 will use C++11.
```

Comment:
Many of the features in C++14 are supported by GCC 4.9:

https://gcc.gnu.org/projects/cxx1y.html
https://gcc.gnu.org/onlinedocs/libstdc++/manual/status.html#status.iso.2014

by the time ROS 2 is released, C++14 will be fully supported by all three major compilers (GCC, Clang, VS2015)